query/cloudfront/cloudfront_distribution_configured_with_origin_failover.sql

```diff
@@ -0,0 +1,16 @@
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when origin_groups ->> 'Items' is not null then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when origin_groups ->> 'Items' is not null then title || ' origin group is configured.'
```

Comment:
The query name and ok/alarm messages are a bit different, does having an origin group configured mean failover is also enabled? Can you please link to the relevant docs on this feature?

Reply:
https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html#fsbp-cloudfront-4
```
This control checks whether an Amazon CloudFront distribution is configured with an origin group that has two or more origins.
```